index.rst

```diff
@@ -948,6 +948,7 @@ Additional Resources
    :hidden:
    :caption: Reinforcement Learning
 
+   beginner/pendulum
```

Comment:
you should move the tutorial to intermediate_source. I don't think it's a beginner tutorial.

Reply:
Also need to add a customcard like for other tutorials. See below.